lib/agent0/agent0/utils.py

```diff
@@ -0,0 +1,12 @@
+"""Various agent0 utility functions."""
```

Comment:
I think we should rename `agent0/test_fixtures` to `agent0/test_utils` and then add a file called `assert_never.py` which contains the contents of this file. So the tree would be
```
agent0/
  > test_utils/
    - __init__.py # includes assert_never
    - assert_never.py # contents of current agent0/utils.py
    - cycle_trade_policy.py # untouched
```